rgw/v2/tests/s3_swift/test_swift_basic_ops.py

```diff
@@ -64,51 +80,79 @@ def test_exec(config):
     user_info = umgmt.create_subuser(tenant_name=tenant, user_id=user_names[0])
     auth = Auth(user_info)
     rgw = auth.do_auth()
+
     for cc in range(config.container_count):
-        container_name = utils.gen_bucket_name_from_userid(user_info['user_id'], rand_no=cc)
-        container = swiftlib.resource_op({'obj': rgw,
-                                          'resource': 'put_container',
-                                          'args': [container_name]})
-        if container is False:
-            raise TestExecError("Resource execution failed: container creation faield")
-        for oc, size in list(config.mapped_sizes.items()):
-            swift_object_name = utils.gen_s3_object_name('%s.container.%s' % (user_names[0], cc), oc)
-            log.info('object name: %s' % swift_object_name)
-            object_path = os.path.join(TEST_DATA_PATH, swift_object_name)
-            log.info('object path: %s' % object_path)
-            data_info = manage_data.io_generator(object_path, size)
-            # upload object
-            if data_info is False:
-                TestExecError("data creation failed")
-            log.info('uploading object: %s' % object_path)
-            with open(object_path, 'r') as fp:
-                rgw.put_object(container_name, swift_object_name,
-                               contents=fp.read(),
-                               content_type='text/plain')
-            # download object
-            swift_object_download_fname = swift_object_name + ".download"
-            log.info('download object name: %s' % swift_object_download_fname)
-            swift_object_download_path = os.path.join(TEST_DATA_PATH, swift_object_download_fname)
-            log.info('download object path: %s' % swift_object_download_path)
-            swift_object_downloaded = rgw.get_object(container_name, swift_object_name)
-            with open(swift_object_download_path, 'w') as fp:
-                fp.write(str(swift_object_downloaded[1]))
-            # modify and re-upload
-            log.info('appending new message to test_data')
-            message_to_append = 'adding new msg after download'
-            fp = open(swift_object_download_path, 'a+')
-            fp.write(message_to_append)
-            fp.close()
-            with open(swift_object_download_path, 'r') as fp:
-                rgw.put_object(container_name, swift_object_name,
-                               contents=fp.read(),
-                               content_type='text/plain')
-            # delete object
-            log.info('deleting swift object')
-            rgw.delete_object(container_name, swift_object_name)
-        # delete container
-        log.info('deleting swift container')
-        rgw.delete_container(container_name)
+        if config.test_ops.get('enable_version', True):
```

Comment:
make the default value as False, so that the existing configs will not change if the variable is not present in the config yaml